app/Factories/PlanetServiceFactory.php

```diff
@@ -443,11 +449,232 @@ private function setupPlanetProperties(Planet $planet): void
         $planet->deuterium = 0;
 
         // Set default mine production percentages to 10 (100%).
-        $planet->metal_mine_percent = 10;
-        $planet->crystal_mine_percent = 10;
-        $planet->deuterium_synthesizer_percent = 10;
-        $planet->solar_plant_percent = 10;
-        $planet->fusion_plant_percent = 10;
-        $planet->solar_satellite_percent = 10;
+        $planet->metal_mine_percent = $planet_data['metal_mine_percent'];
+        $planet->crystal_mine_percent = $planet_data['crystal_mine_percent'];
+        $planet->deuterium_synthesizer_percent = $planet_data['deuterium_synthesizer_percent'];
+        $planet->solar_plant_percent = $planet_data['solar_plant_percent'];
+        $planet->fusion_plant_percent = $planet_data['fusion_plant_percent'];
+        $planet->solar_satellite_percent = $planet_data['solar_satellite_percent'];
```

Comment:
Currently in OGameX the `*_percent` fields in the `planets` table are associated with the production percentage that the player can configure on the resource settings page, example: 
<img width="670" alt="Screenshot 2024-12-23 at 21 30 47" src="https://github.com/user-attachments/assets/01c995f2-fe20-4ed8-9d09-95af8d9ab2a1" />

If I read your code correctly then based on the position of the planet a bonus modifier should be added to the mine output. However to make this work we'll probably need to add this bonus modifier to the resource production formula. See `PlanetService::updateResourceProductionStatsInner()` for how this is currently implemented.

So for this PR I would propose to:
1. Revert this specific change back to the original version (set everything to "10" which represents 100% in the screenshot above)
2. Integrate the planet resource bonus modifier into the `PlanetService::updateResourceProductionStatsInner()` logic. However as this is outside of the scope of #391 we could also make a new issue for this to take on separately.